pg_nearest_city/scripts/voronoi_generator.py

```diff
@@ -496,6 +753,18 @@ def _verify_output_files(self):
         self.logger.info(f"  - {cities_file}")
         self.logger.info(f"  - {voronoi_file}")
 
+    def _vacuum_full_and_analyze_db(self, conn):
```

Comment:
This was added due to repeated runs causing bloat in the DB, as Postgres' default behavior for autovacuum won't kick in until far more dead tuples exist than we're likely to add.

**WARNING**: `VACUUM FULL` locks the table while running (on my Mac, it takes ~10 seconds to complete both `country` and `geocoding`), but I assume if you're actively importing data, there won't be traffic hitting it? If there is, this can be removed, and another method found; or this can be scheduled to run during off-peak times.